Modules/PrinterManager/Windows/WindowsPrinterManager.cs

```diff
@@ -131,8 +132,38 @@ public override void Configure(Printer printer, bool verbose = false)
 
             if (string.IsNullOrEmpty(printer.ConfigFile)) return;
 
-            Log.Entry(LogName, "Configuring " + printer.Name);
-            PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f g p", verbose);
+            if (File.Exists(printer.ConfigFile))
+            {
+                Log.Entry(LogName, "Configuring " + printer.Name);
+                PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f g p", verbose);
+                PrintUI($"/Sr /n \"{printer.Name}\" /a \"{printer.ConfigFile}\" m f u p", verbose);
+            }
+            else {
```

Comment:
Wouldn't mind if we can stay with the coding style - curly brackets in new lines.

Reply:
Updated :)